docs/index.html

```diff
@@ -458,7 +458,16 @@ <h1 id="api-commonformats">Common Formats</h1>
 // $dt->toAtomString() is the same as $dt->format(DateTime::ATOM);
 echo $dt->toAtomString();      // 1975-12-25T14:15:16-05:00
 echo $dt->toCookieString();    // Thursday, 25-Dec-1975 14:15:16 EST
+
 echo $dt->toIso8601String();   // 1975-12-25T14:15:16-05:00
+// Be aware we chose to use the full-extended format of the ISO 8601 norm
+// Natively, DateTime::ISO8601 format is not compatible with ISO-8601 as it
+// is explained here in the PHP documentation:
+// http://php.net/manual/fr/class.datetime.php#datetime.constants.iso8601
```

Comment:
links to french documentation. also http?
`https://php.net/manual/class.datetime.php#datetime.constants.iso8601` should work and be language agnostic.

Reply:
Sorry for that, thanks, I will fix it.